src/web/config_load.cc

```diff
@@ -31,7 +31,6 @@
 #include "config/client_config.h"
 #include "config/config_definition.h"
 #include "config/config_setup.h"
-#include "config/directory_tweak.h"
```

Comment:
It's needed for https://github.com/gerbera/gerbera/blob/e3a66e274fdb7f4de46d877fbf3579c4e273f43e/src/web/config_load.cc#L210

May happen that some other include get's it.

Reply:
That's probably what's happening.

Reply:
So we should keep it for clarity (others are there as well)